engines/efh/efh.cpp

```diff
@@ -826,49 +829,52 @@ void EfhEngine::handleWinSequence() {
 		getInput(1);
 	}
 
-	Common::KeyCode input = Common::KEYCODE_INVALID;
+	_customAction = kActionNone;
+	setKeymap(kKeymapSkipVideo);
 
-	while (input != Common::KEYCODE_ESCAPE && !shouldQuit()) {
+	while (_customAction != kActionSkipVideo && !shouldQuit()) {
 		displayRawDataAtPos(winSeqSubFilesArray1[0], 0, 0);
 		displayFctFullScreen();
 		displayRawDataAtPos(winSeqSubFilesArray1[0], 0, 0);
-		input = getInput(32);
-		if (input != Common::KEYCODE_ESCAPE) {
+		getInput(32);
```

Comment:
I do not like this function with side-effect, e.g. that it changes global state.

Instead, please keep it just as it was, e.g.:
```C++
intpu = getInput(32);

if (input != kActionSkipVideo)
```
etc.

Just keep it changing `_customAction` as it does now, but also return this value for code readability.